docs/content/usage-reporting.md

```diff
@@ -0,0 +1,217 @@
+    ```
+    > echo -n 'foo' | base64
+    Zm9v
+    > echo -n 'bar' | base64
+    YmFy
+    ```
+
+2. In order to make the credential available to NGINX Cluster Connector, we need to create a Kubernetes secret. Copying the following content to a text editor, and fill in the username and password under `data`, with base64 representation of the username and password obtained in step 1:
+    ```
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: nms-basic-auth
+      namespace: nginx-cluster-connector
+    type: kubernetes.io/basic-auth
+    data:
+      username: Zm9v # base64 representation of 'foo' obtained in step 1
+      password: YmFy # base64 representation of 'bar' obtained in step 1
+    ```
+    Save this in a file named `nms-basic-auth.yaml`. In the example, the namespace is `nginx-cluster-connector` and the secret name is `nms-basic-auth`. The namespace is the default namespace for the NGINX Cluster Connector. If you are using a different namespace, please change the namespace in the `metadata` section of the file. Note that the cluster connector only supports basic-auth secret type in `data` format, not `stringData`, with the username and password encoded in base64.
+3. If the namespace `nginx-cluster-connector` does not exist, create the namespace:
+    ```
+    > kubectl create namespace nginx-cluster-connector
+    ```
+4. Deploy the Kubernetes secret created in step 2 to the Kubernetes cluster:
+    ```
+    > kubectl apply -f nms-basic-auth.yaml
+    ```
+   If you want to update the basic-auth credentials to NMS server in the future, update the `username` and `password` fields, and apply the changes by running the command again. The NGINX Cluster Connector will automatically detect the changes and use the new username and password without redeploying the NGINX Cluster Connector.
+
+5. Download the deployment file [nginx-cluster.yaml](https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/v3.1.1/examples/shared-examples/usage-reporting/cluster-connector.yaml). Edit the following under `args` section
+   ```
+        args:
+        - -nms-server-address=https://nms.example.com/api/platform/v1/k8s-usage
+        - -nms-basic-auth-secret=nginx-cluster-connector/nms-basic-auth
+   ```
+   The `-nms-server-address` should be the address of the NGINX Management Suite host. The `nms-basic-auth-secret` should be the namespace/name of the secret created in step 2, i.e `nginx-cluster-connector/nms-basic-auth`.
+   For more information on the command-line flags, see [Configuration](#configuration).
+
+6. To deploy the Nginx Cluster Connector, save the file above and run the following command to deploy it to your Kubernetes cluster:
+   ```
+   > kubectl apply -f cluster-connector.yaml
+   ```
```

Comment:
Where is the image location connected?
looking at: examples/shared-examples/usage-reporting/cluster-connector.yaml
it looks like it is assuming the image location is known by the cluster or the image is already local.  There is no path to registry.nginx.com/cluster-connector 

This should pull from the public repo location by default.